### PR TITLE
Update en.yml

### PR DESCRIPTION
**Pull request by nnunn**
Modifying bpsc_read_on_site_requests message as per Linda Quirk's helpdesk ticket https://helpdesk.library.ualberta.ca/otrs/index.pl?Action=AgentTicketZoom;TicketID=60275.

## Context

Minor text change to remove the need to continually update the message.

https://helpdesk.library.ualberta.ca/otrs/index.pl?Action=AgentTicketZoom;TicketID=60275

## What's New

Removed mention of Fall 2020 term

- [ ] `rake spec` tests pass

Remember to request a reviewer when you submit!

### CODE REVIEW

**Review by pgwillia — CHANGES_REQUESTED** [review]
Looks good.  Just need to add a [Changelog entry](https://github.com/ualbertalib/discovery/blob/bpsc-text-update/CHANGELOG.md)

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ and releases in Discovery project adheres to [Semantic Versioning](http://semver
 
 ## [Unreleased]
 
+2021-03-19
```
Comment (pgwillia) [inline CHANGELOG.md:9]: Do you want to cut a release right away?

Then you could do:
```suggestion
## [3.5.9] - 2021-03-19
```
And update https://github.com/ualbertalib/discovery/blob/d7acb7c92dba33ed19a291440e15c3a43496a2e3/config/application.rb#L11 with 3.5.9 too.

Comment (nnunn) [reply CHANGELOG.md:9]: I don't need to cut a release - or do I? I'm fine with this change just being part of the next release as it is so small.

Comment (pgwillia) [reply CHANGELOG.md:9]: No you don't need to cut a release.  If that's not your intention this line doesn't really need to be there at all.

Comment (nnunn) [reply CHANGELOG.md]: Is is right now? sorry - I'm pretty rusty at this :)!

**Pull request merged**